cluster/ci/config/prow/config.yaml

```diff
@@ -2814,6 +2814,55 @@ presubmits:
         - --artifact-dir=$(ARTIFACTS)
         - --target=unit
         - --target=[images]
+  - name: pull-ci-kubernetes-descheduler-e2e-gce
+    agent: kubernetes
+    always_run: true
+    context: ci/prow/e2e
+    rerun_command: "/test e2e"
+    trigger: "((?m)^/test( all| e2e),?(\\s+|$))"
+    decorate: true
+    skip_cloning: true
+    spec:
+      serviceAccountName: ci-operator
+      volumes:
+      - name: job-definition
+        configMap:
+          name: prow-job-cluster-launch-e2e
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-gcp
+          - configMap:
+              name: cluster-profile-gcp
+      containers:
+      - name: test
+        image: ci-operator:latest
+        volumeMounts:
+        - name: job-definition
+          mountPath: /usr/local/e2e-gcp
+          subPath: cluster-launch-e2e.yaml
+        - name: cluster-profile
+          mountPath: /usr/local/e2e-gcp-cluster-profile
+        env:
+        - name: JOB_NAME_SAFE
+          value: kubernetes-descheduler-e2e
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              name: ci-operator-kubernetes-descheduler
+              key: e2e.json
+        - name: TEST_COMMAND
+          value: "make test-e2e"
+        command:
+        - ci-operator
+        args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --secret-dir=/usr/local/e2e-gcp-cluster-profile
+        - --template=/usr/local/e2e-gcp
+        - --target=e2e-gcp
```

Comment:
won't this run the origin unit tests?

Reply:
I might be misunderstanding and/or abusing it, but setting this enviroment
variable should run the e2e tests:

https://github.com/openshift/release/blob/master/cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml#L133-L137